testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -698,6 +787,9 @@ public void testCreateTableLike()
         assertThat(event.getIoMetadata().getOutput().get().getSchema()).isEqualTo("default");
         assertThat(event.getIoMetadata().getOutput().get().getTable()).isEqualTo("create_simple_table");
         assertThat(event.getIoMetadata().getOutput().get().getColumns().get())
-                .containsExactly("test_column", "test_varchar", "test_bigint");
+                .containsExactly(
+                        new OutputColumnMetadata("test_column", ImmutableSet.of()),
+                        new OutputColumnMetadata("test_varchar", ImmutableSet.of()),
+                        new OutputColumnMetadata("test_bigint", ImmutableSet.of()));
```

Comment:
Please add tests with column references in:
- `WITH` query
- aggregation's filter, aggregation's ORDER BY
- ORDER BY
- WHERE

A question: why are the grouping columns of an aggregation not recorded?